man/systemd-keyutil.xml

```diff
@@ -64,6 +64,16 @@
 
         <xi:include href="version-info.xml" xpointer="v257"/></listitem>
       </varlistentry>
+
+      <varlistentry>
+        <term><command>pkcs7</command></term>
+
+        <listitem><para>This command generates a PKCS#7 signature that embeds the PKCS#1 signature (RSA)
+        provided with <option>--signature=</option> to an <option>--output=</option> file in PKCS#7 format
+        (p7s) using the certificate given with <option>--certificate=</option>.</para>
```

Comment:
hmm, the "to" is a bit weird, not sure if it refers to the "generates" or the "embeds"; but in both cases sounds weird